modules/aerospike/aerospike_test.go

```diff
@@ -0,0 +1,64 @@
+package aerospike_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	as "github.com/testcontainers/testcontainers-go/modules/aerospike"
+)
+
+const (
+	aerospikeImage = "aerospike/aerospike-server:latest"
+)
+
+// TestAerospike tests the Aerospike container functionality
+// It includes tests for starting the container with a valid image,
+// applying container customizations, and handling context cancellation.
+// It also includes a test for an invalid image to ensure proper error handling.
+// The tests use the testcontainers-go library to manage container lifecycle
+func TestAeroSpike(t *testing.T) {
+	t.Run("fails_with_invalid_image", func(t *testing.T) {
+		ctx := context.Background()
+		_, err := as.Run(ctx, "invalid-aerospike-image")
+		require.Error(t, err)
+		require.Contains(t, err.Error(), "failed to start Aerospike container")
+	})
+
+	t.Run("succeeds_with_valid_image", func(t *testing.T) {
+		ctx := context.Background()
+		container, err := as.Run(ctx, aerospikeImage)
+		require.NoError(t, err)
+		require.NotNil(t, container)
+		defer container.Container.Terminate(ctx)
+
+		host, err := container.GetHost()
+		require.NoError(t, err)
+
+		port, err := container.GetPort()
+		require.NoError(t, err)
+
+		require.NotEmpty(t, host)
+		require.NotEmpty(t, port)
```

Comment:
suggestion: if the methods are removed, then let's remove this too, as it's testing the library code